functions/install/bedrock.sh

```
@@ -0,0 +1,72 @@
+    bin_bytes=$(stat -c%s bedrock_server 2>/dev/null || stat -f%z bedrock_server 2>/dev/null)
+    bin_size=$(printf "%.2f MB" $((bin_bytes / 1000000)))
+    printout info "Server binary downloaded successfully (Size: $bin_size)"
+    create_config "mc_bedrock_vanilla"
+    launchBedrockVanillaServer
+    exit
+}
+
+
+function install_pmmp {
+    printout info "Starting installation of PocketMineMP..."
+    cd $HOME
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)